### PR TITLE
Add test suite of autoyast_mini for s390x and aarch64

### PR DESCRIPTION
**Pull request by coolgw**
Add test suite of autoyast_mini for s390x and aarch64

- Related ticket: https://progress.opensuse.org/issues/131342
- Needles: na
- Verification run: 
    s390x:   https://openqa.suse.de/t11614211
    aarch64: https://openqa.suse.de/tests/11609007#step/verify_cloned_profile/6

### CODE REVIEW

File: test_data/yast/autoyast/profiles/mini_s390x.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (jknphy) [inline test_data/yast/autoyast/profiles/mini_s390x.yaml:1]: this file looks repeated comparing with the one above.

**Review by jknphy — COMMENTED** [review]
Verifications do not match the schedules.
Is it possible to use the same schedule for both cases?

**Comment by coolgw** [comment]
> Verifications do not match the schedules. Is it possible to use the same schedule for both cases?

Verification is use clone command which clone base on create_gnome test cases, not by trigger command, do you mean trigger case in yast group?

./mycustom_clone_script autoyast_mini 11162311 \
yaml_schedule=schedule/yast/autoyast/autoyast_mini_aarch64.yaml \
autoyast=autoyast_sle15/mini.xml \
publish_hdd_1='' \
publish_pflash_vars=''

./mycustom_clone_script autoyast_mini 11162444 \
yaml_schedule=schedule/yast/autoyast/autoyast_mini_s390x.yaml \
autoyast=autoyast_sle15/mini_s390x.xml \
publish_hdd_1='' \
publish_pflash_vars='"

File: test_data/yast/autoyast/profiles/mini_s390x.yaml (hunk 1 above)
Comment (coolgw) [reply test_data/yast/autoyast/profiles/mini_s390x.yaml:1]: Since i have no idea how to make good name for this file, maybe mini_aarch64_s390x.yaml?

**Comment by jknphy** [comment]
> > Verifications do not match the schedules. Is it possible to use the same schedule for both cases?
> 
> Verification is use clone command which clone base on create_gnome test cases, not by trigger command, do you mean trigger case in yast group?
> 
> ./mycustom_clone_script autoyast_mini 11162311 yaml_schedule=schedule/yast/autoyast/autoyast_mini_aarch64.yaml autoyast=autoyast_sle15/mini.xml publish_hdd_1='' publish_pflash_vars=''
> 
> ./mycustom_clone_script autoyast_mini 11162444 yaml_schedule=schedule/yast/autoyast/autoyast_mini_s390x.yaml autoyast=autoyast_sle15/mini_s390x.xml publish_hdd_1='' publish_pflash_vars='"

your VRs do not include modules like `- autoyast/verify_cloned_profile` for instance, seems that you run it with the basic modules.

**Comment by coolgw** [comment]
> > > Verifications do not match the schedules. Is it possible to use the same schedule for both cases?
> > 
> > 
> > Verification is use clone command which clone base on create_gnome test cases, not by trigger command, do you mean trigger case in yast group?
> > ./mycustom_clone_script autoyast_mini 11162311 yaml_schedule=schedule/yast/autoyast/autoyast_mini_aarch64.yaml autoyast=autoyast_sle15/mini.xml publish_hdd_1='' publish_pflash_vars=''
> > ./mycustom_clone_script autoyast_mini 11162444 yaml_schedule=schedule/yast/autoyast/autoyast_mini_s390x.yaml autoyast=autoyast_sle15/mini_s390x.xml publish_hdd_1='' publish_pflash_vars='"
> 
> your VRs do not include modules like `- autoyast/verify_cloned_profile` for instance, seems that you run it with the basic modules.

My mistake,  update the result, but s390x seems failed on verify_cloned_profile, need further investigate.

**Comment by coolgw** [comment]
> > > > Verifications do not match the schedules. Is it possible to use the same schedule for both cases?
> > > 
> > > 
> > > Verification is use clone command which clone base on create_gnome test cases, not by trigger command, do you mean trigger case in yast group?
> > > ./mycustom_clone_script autoyast_mini 11162311 yaml_schedule=schedule/yast/autoyast/autoyast_mini_aarch64.yaml autoyast=autoyast_sle15/mini.xml publish_hdd_1='' publish_pflash_vars=''
> > > ./mycustom_clone_script autoyast_mini 11162444 yaml_schedule=schedule/yast/autoyast/autoyast_mini_s390x.yaml autoyast=autoyast_sle15/mini_s390x.xml publish_hdd_1='' publish_pflash_vars='"
> > 
> > 
> > your VRs do not include modules like `- autoyast/verify_cloned_profile` for instance, seems that you run it with the basic modules.
> 
> My mistake, update the result, but s390x seems failed on verify_cloned_profile, need further investigate.

We need some small update for s390x so now we need two files.

**Pull request merged**